### PR TITLE
Create newtalk.tw.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
image
image

**Comment by gloomy-ghost** [comment]
Timeout
admin.newtalk.tw

InvalidCert
www.newtalk.tw
3fe8l33fe8f3ff0f0oc3ff07fa0al2000.newtalk.tw
www.f38lo3f3fe8e3ff08c33fe8f3ff0f0al.newtalk.tw
l33fe8f34730ff0f896c0oc3ff0al.newtalk.tw
l3f7fd83ff0f0oc3ff07fa0al2000.newtalk.tw
lo3ff07fd83fe8c3fe83f46e13ff0f3fe808061al.newtalk.tw
lo47fd8e53c3fe83ff0al.newtalk.tw
lo47fd8e53c3ff0a3ff0l.newtalk.tw
lo73fe83ff0fd83fe8c3fe83f3ff0f0a3fe8l.newtalk.tw
www.lo73ff0f3ff0d83fe8c3fe83f3ff0f0a3fe8l.newtalk.tw
lo7f3ff0d83fe8c3fe83f3f7fd8f0f0al.newtalk.tw
lo7fd3fe883fe8c3f3ff0f0al.newtalk.tw
loa047fd8e53c3ff0a1000l.newtalk.tw
local.newtalk.tw
m.newtalk.tw
mail.newtalk.tw
www1.newtalk.tw

**Comment by gloomy-ghost** [comment]
`www` is in SANs now.

Also found a CDN domain: https://s.ntkstatic.org/media/blog/293/blog_img_293.jpg

**Comment by gloomy-ghost** [comment]
话说那坨奇怪的域名好像都重定向到`^`了

**Comment by ivysrono** [comment]
试了下，那一大坨奇怪域名都SANs了，http会重定向到^,https就不跳转了。
这个怎么处理？直接https？

**Comment by gloomy-ghost** [comment]
还是有不在的，比如说 https://www.ssllabs.com/ssltest/analyze.html?d=lo3ff07fd83fe8c3fe83f46e13ff0f3fe808061al.newtalk.tw

我觉得注释里说一下这是泛解析，然后把这些都直接重定向到`^`吧…

**Comment by ivysrono** [comment]
大爷的，还发现了坨无限域名：nvk\d*.
我还要去处理渣浪的……

**Comment by gloomy-ghost** [comment]
其实是 `.+\.`

**Comment by ivysrono** [comment]
admin.newtalk.tw
local.newtalk.tw
m.newtalk.tw
不知道这几个有没有特殊意义……

**Comment by ivysrono** [comment]
见鬼，过去7天才看到这条评论……

File: src/chrome/content/rules/newtalk.tw.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+<ruleset name="newtalk.tw">
+
+	<!--	Complications:	-->
+	<target host="*.newtalk.tw" />
+		<test url="http://abcdefg123456789.newtalk.tw/" />
+		<test url="http://ap1.newtalk.tw/" />
+		<test url="http://www.newtalk.tw/" />
+
+	<rule from="^http://\w+\.newtalk\.tw/" to="https://newtalk.tw/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/newtalk.tw.xml]: `\w+` doesn't match with dash, such as abcdefg-123456789.newtalk.tw. I'd suggest you to use `.+`.

Comment (ivysrono) [inline src/chrome/content/rules/newtalk.tw.xml]: 我就记得哪次中招过，`.+` 通不过测试

Comment (gloomy-ghost) [inline src/chrome/content/rules/newtalk.tw.xml]: `.+` 会要求两个额外的测试

Comment (ivysrono) [inline src/chrome/content/rules/newtalk.tw.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/185161024#L193
> failure: /opt/utils/../src/chrome/content/rules/newtalk.tw.xml failed test: The 'from' rule contains unescaped period in regular expression.  Try escaping it with a backslash.
这个是多加两个测试就会通过？

Comment (gloomy-ghost) [inline src/chrome/content/rules/newtalk.tw.xml]: 丧心病狂的电信……直接用手机开热点了

之前还没注意过[/utils/trivial-validate.py#L74-L84](https://github.com/EFForg/https-everywhere/blob/445df151702555218cda70aab9aff092f1bfd214/utils/trivial-validate.py#L74-L84)，看了下觉得这个检测方式的误报率有点高……

留着`\S`吧，顺手 git blame Yimg.jp.xml （逃

**Pull request merged**